src/gt4py/next/ffront/past_to_itir_wf.py

```diff
@@ -0,0 +1,57 @@
+# GT4Py - GridTools Framework
```

Comment:
Because that one is already huge and scrolling past the visitor is annoying. I could imagine something like

```
src/
  gt4py/
    next/
      ffront/
        past_to_itir/
          __init__.py
          visitor.py
          workflow.py  <-- in here?
```

Reply:
My view would be: make `ProgramLowering` private, then the only visible thing is the workflow. Personally, I wouldn't add an extra package.

Maybe, in a future refactoring, we could add a layer (maybe decorator) that makes an eve visitor a workflow. Currently we follow (in most places) the convention that a visitor/transformation has an apply which runs the transformation. The extra work in past_to_itir_wf could be part of the apply and the decorator would just turn `apply()` -> `__call__`. Just ideas.

Your choice how to leave the structure in this PR.

Reply:
I will pack it into the same module. I will not make the additional work in that step part of `ProgramLowering.apply()` in this PR because I can not judge if it would break a significant separation of concerns .
Also, for now I will refrain from renaming `ProgramLowering` for the following reasons:
- We have the convention that implementation details do not need to but are allowed to start with an underscore
- Importing or using `_ProgramLowering` in it's unit tests makes it seem like there is something tested that probably shouldn't be
- If after checking it turns out that the extra work required for the workflow step should indeed be part of `ProgramLowering`, then it would have to be undone again anyway.